test/test_logger_initializer.rb

```diff
@@ -5,11 +5,8 @@
 class LoggerInitializerTest < ::Test::Unit::TestCase
   TMP_DIR = File.expand_path(File.dirname(__FILE__) + "/tmp/logger_initializer#{ENV['TEST_ENV_NUMBER']}")
 
-  teardown do
-    begin
-      FileUtils.rm_rf(TMP_DIR)
-    rescue => _
-    end
+  setup do
+    FileUtils.rm_rf(TMP_DIR) rescue nil
```

Comment:
> to allow us to confirm the data after executing a test.

Is it really needed?
Generally speaking tmp files shouldn't exist after test so that it should be removed at `teardown`.
(It's OK to do in also `setup` to make sure.)

Reply:
Oh, I see.
I thought that tests should not leave tmp data outside the working directory, but can leave them inside.

Indeed, many other tests delete tmp files at `teardown`.

Reply:
> I thought that tests should not leave tmp data outside the working directory, but can leave them inside.

I found out that what I was thinking was wrong! I learned!

I will fix `setup`! Thanks!

Reply:
I have fixed this to make sure not to leave tmp files.